org.eclipse.scanning.test/src/org/eclipse/scanning/test/command/ScanRequestCreationTest.java

```diff
@@ -295,7 +332,7 @@ public void testRepeatCommandWithMonitorsNoDetector() throws Exception {
 	}
 
 	@Test
-	public void testRasterCommandWithROIs() throws Exception {
+	public void testRasterCommandWithROIs() throws GeneratorException {
 		pi.exec("sr =                                 "
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "sr =                                 " 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)